### DISCUSSION TITLE
Suggestions for 1OO

### DISCUSSION THREAD

**Discussion by joaogui1** [fogus/thunks#2] [Ideas]
Just adding some suggestions for your list for 100 OOP books:
- [99 bottles of OOP](https://sandimetz.com/99bottles) (people often recommend the Ruby version)
- [Eloquent Ruby](http://eloquentruby.com/) 
- [Practical Object Oriented Design in Ruby](https://www.amazon.com/gp/product/0321721330/)
- [Fluent Python](https://www.amazon.com/Fluent-Python-Concise-Effective-Programming/dp/1492056359/ref=pd_lpo_1?pd_rd_i=1492056359&psc=1) (arguably a python book, but in python pretty much everything is an object)
- [Object-Oriented Programming with SIMULA](https://dl.acm.org/doi/10.5555/534000)
- [Self Handbook](https://handbook.selflanguage.org/2017.1/)
- Not a book, but this [talk](https://www.youtube.com/watch?v=2szKoUQoNm8) on Prototypes and OOP in Scheme

**Comment by fogus** [comment]
The Self Handbook is a must!
